controller-server/types/dancer.d.ts

```diff
@@ -0,0 +1,10 @@
+enum DancerName {
```

Comment:
Why the enum here?

Reply:
We only need to change the value inside the enum, use enum is more meaningful than simply string.